### PR TITLE
Update optional-claims-reference.md to mention the optional value on idtype

### PR DESCRIPTION
**Pull request by jmprieur**
@SHERMANOUKO this will also need to be propagated to 1P, ?What is your preference? shall I do another PR?

### CODE REVIEW

File: docs/identity-platform/optional-claims-reference.md [hunk 1]
```diff
@@ -100,6 +100,8 @@ Some optional claims can be configured to change the way the claim is returned.
...
 | | `use_guid` | Emits the client ID of the resource (API) in GUID format as the `aud` claim always instead of it being runtime dependent. For example, if a resource sets this flag, and its client ID is `bb0a297b-6a42-4a55-ac40-09a501456577`, any app that requests an access token for that resource receives an access token with `aud` : `bb0a297b-6a42-4a55-ac40-09a501456577`. Without this claim set, an API could get tokens with an `aud` claim of `api://MyApi.com`, `api://MyApi.com/`, `api://myapi.com/AdditionalRegisteredField` or any other value set as an app ID URI for that API, and the client ID of the resource. |
+| `idtyp` | | This claim is used to get the type of token (app, user, device). By default it's only emitted for app-only tokens. Like all optional claims that affect the access token, the resource in the request must set this optional claim, since resources own the access token. |
+| | `include_user_token` | Emits the `idtyp` claim for users token. Without this optional additional peroperty for the idtyp claim set, an API only gets the claim for app tokens. |
```
Comment (keegan-caruso) [inline docs/identity-platform/optional-claims-reference.md:104]: typo "peroperty" => "property"

**Review by keegan-caruso — APPROVED** [review]

**Comment by jmprieur** [comment]
#sign-off

**Comment by Court72** [comment]
Hi @jmprieur - This pull request was opened in the public repo. PMs should work in the private repo, per the Microsoft Docs contributor guide. Would you make this content update in the private repo instead? Thank you! We'll close this PR.

**Comment by cilwerner** [comment]
@jmprieur, I've fixed this!

**Pull request closed without merging**